carapace-server/src/main/java/org/carapaceproxy/api/CertificatesResource.java

```diff
@@ -152,32 +152,12 @@ private CertificateBean findCertificateById(String certId) {
 
             if (certificate.isDynamic()) {
                 DynamicCertificateState state = server.getDynamicCertificateManager().getStateOfCertificate(certBean.getId());
-                certBean.setStatus(stateToStatusString(state));
+                certBean.setStatus(state.toString());
             }
             return certBean;
         }
 
         return null;
     }
 
-    static String stateToStatusString(DynamicCertificateState state) {
```

Comment:
Why?
This is important, in order to have a clear definition of what is presented in the API